portal/templates/redesign/help-and-support_new.html

```diff
@@ -8,13 +8,33 @@
 
 {% block scripts %}
 {{ block.super }}
+<script>
+    $( document ).ready(function() {
+        questions = $('.faqs').find('div').each(function() {
+            $( this ).children('p').html('<span>+　</span>'+$( this ).children('p').html())
```

Comment:
This actually targets all the children of the faqs div which are p elements. Because of this, the + sign is added next to the answer text as well :P
A way to solve this would be to add a class to the paragraphs containing the questions, and target them through this class name.